apps/client/src/components/RegisterLookupSummary.js

```diff
@@ -42,16 +42,18 @@ const RegisterLookupSummary = ({
     <ComponentWrapper marginBottom={hasIMTR ? "0" : null}>
       <AddressLines
         {...address}
+        editAddressRenderer={() =>
+          compact ? null : (
```

Comment:
why not do it like !compact && <ChangeAddressModal {...{ hasIMTR, setActiveState, slug }} />
imo it's easier to read. Than compact ? null :

Reply:
good point, fixed it